### DISCUSSION TITLE
React native blur view on new arch

### DISCUSSION THREAD

**Discussion by TheRogue76** [reactwg/react-native-new-architecture#47] [Libraries]
I’m starting this discussion because the old blur library hasn’t had an update in a long time and it might be a good idea to update it for the new RNs and the new arch (or potentially replace it with something like https://github.com/idrisssakhi/blurView)

**Comment by TheRogue76** [comment]
@idrissakhi

**Comment by chakrihacker** [comment]
I can pick this

**Reply by TheRogue76** [reply]
Thanks. Do you plan to maintain the old one or to replace it with a new implementation?

**Reply by chakrihacker** [reply]
I don't have plans to maintain old one,

But can implement and maintain new arch

**Comment by gispada** [comment]
If you are referring to [react-native-blur](https://github.com/Kureev/react-native-blur), I started the migration and I'm about 50% done (iOS seems ok, starting Android). I'm not a maintainer of the library, but I plan to make a PR at the end.
